public/js/test/mochitest/head.js

```diff
@@ -58,6 +87,17 @@ function _afterDispatchDone(store, type) {
   });
 }
 
+/**
+ * Wait for a specific action type to be dispatch.
+ * If an async action, will wait for it to be done.
+ *
+ * @memberof mochitest/waits
+ * @param {Object} dbg
+ * @param {String} type
+ * @param {Number} eventRepeat
+ * @return {Promise}
+ * @static
+ */
```

Comment:
That's a good point, @jbhoosreddy. We don't compile these files so we can't use Flow here, because there's no way to strip the types. We probably have to conform to this syntax. Great job!